ci_scripts/install.sh

```diff
@@ -37,11 +37,18 @@ python -c "import numpy; print('numpy %s' % numpy.__version__)"
 python -c "import scipy; print('scipy %s' % scipy.__version__)"
 
 if [[ "$EXAMPLES" == "true" ]]; then
+    conda install --yes gxx_linux-64 gcc_linux-64 swig
     pip install -e '.[examples]'
+    pip install -e '.[examples_unix]'
 fi
 if [[ "$DOCTEST" == "true" ]]; then
     pip install sphinx_bootstrap_theme
 fi
+if [[ "$DOCPUSH" == "true" ]]; then
+    conda install --yes gxx_linux-64 gcc_linux-64 swig
+    pip install -e '.[examples]'
+    pip install -e '.[examples_unix]'
```

Comment:
You can save a line by having both in a single statement: `pip install -e '.[examples,examples_unix]'`.